pytorch/training/docker/2.1/py3/cu121/Dockerfile.gpu

```diff
@@ -0,0 +1,321 @@
+ARG PYTHON=python3
```

Comment:
some comments from cu118 dockerfile also apply to this one

Reply:
fixed alongside cu118 file in [af40080](https://github.com/aws/deep-learning-containers/pull/3389/commits/af4008083c72013e091f6da1ee0aa62117361752)